projects/rocprofiler-sdk/source/lib/python/rocpd/libpyrocpd.cpp

```diff
@@ -419,8 +418,112 @@ PYBIND11_MODULE(libpyrocpd, pyrocpd)
                 fmt::format("Perfetto generation from {} SQL database(s)", data.size())};
             for(auto obj : {data.connection})
             {
-                auto* conn  = rocpd::interop::get_connection(std::move(obj));
-                auto  nodes = rocpd::read<rocpd::types::node>(conn);
+                auto* conn = rocpd::interop::get_connection(std::move(obj));
+
+                const std::string create_samples_view = R"(
+                            CREATE TEMP VIEW
```

Comment:
```suggestion
                            CREATE TEMP VIEW IF NOT EXISTS
```
When trying to convert 2 databases, this errors out because the view already exits.  Same with the next view.

Reply:
Done